posthog-core/src/index.ts

```diff
@@ -93,6 +93,10 @@ export abstract class PostHogCoreStateless {
     }
     this.requestTimeout = options?.requestTimeout ?? 10000 // 10 seconds
     this.disableGeoip = options?.disableGeoip ?? true
+
+    if (options?.__onConstructed) {
+      options.__onConstructed(this)
+    }
```

Comment:
This feels like a hack and I'm not sure what it is providing different from awaiting the SDK via initAsync? At some point someone will see this and use it, even though its meant to be "just for testing". 
Is there no alternative?

Reply:
It's now gone, I do call a private method in test code now though